.gitignore

```diff
@@ -10,3 +10,5 @@ sounds/*
 !sounds/.keep
 
 db.json
+
+*.swp
```

Comment:
I don't think this is necessary.

Reply:
I use vim often so I added this without thinking - I can remove it from here if you would prefer that.

Reply:
Yes please! 🙂